src/expr/node.h

```diff
@@ -1331,6 +1360,117 @@ TypeNode NodeTemplate<ref_count>::getType(bool check) const
   return NodeManager::currentNM()->getType(*this, check);
 }
 
+template <bool ref_count>
+inline Node NodeTemplate<ref_count>::substituteCaptureAvoiding(
```

Comment:
I wouldn't use `inline` here. If it is here for performance reasons, the compiler should automatically figure out whether to inline the code or not and we probably don't need it for linking reasons here either.